src/dodal/devices/fast_grid_scan.py

```diff
@@ -181,181 +193,153 @@ def non_integer_dwell_time(cls, dwell_time_ms: float) -> float:
         return dwell_time_ms
 
 
-class GridScanCompleteStatus(DeviceStatus):
+class PandAGridScanParams(GridScanParamsCommon):
```

Comment:
It might be nicer if I separate this file into two. One for the params and one for the devices

Reply:
I'll do this in a separate PR, since that will be annoying to review combined with this